tests/scripts/travis_setup.sh

```diff
@@ -25,25 +25,43 @@ export CATALINA_HOME='.'
 export JAVA_OPTS="-Xms1024m -Xmx1024m -XX:MaxPermSize=512m -XX:+CMSClassUnloadingEnabled -Djavax.net.ssl.trustStore=$CATALINA_HOME/fedora/server/truststore -Djavax.net.ssl.trustStorePassword=tomcat"
 # TODO: roll a Fedora 3.8.1 islandora_tomcat that doesn't require a rebuild.
 if [ $FEDORA_VERSION = "3.8.1" ]; then
-  export FEDORA_HOME=fedora
+  export FEDORA_HOME=$HOME/islandora_tomcat/fedora
   ./fedora/server/bin/fedora-rebuild.sh -r org.fcrepo.server.utilities.rebuild.SQLRebuilder
 fi
 ./bin/startup.sh
 
-# Drush installation.
 cd $HOME
-pear channel-discover pear.drush.org
-pear upgrade --force Console_Getopt
-pear upgrade --force pear
-pear channel-discover pear.drush.org
-wget http://alpha.library.yorku.ca/drush-6.3.tar.gz
-tar xf drush-6.3.tar.gz
-sudo mv drush-6.3 /opt/
-sudo ln -s /opt/drush-6.3/drush /usr/bin/drush
 
-# PHPCS installation.
-wget http://alpha.library.yorku.ca/PHP_CodeSniffer-1.5.6.tgz
-pear install PHP_CodeSniffer-1.5.6.tgz
+# Drush and PHPCS installation.
+if [ "$(phpenv version-name)" == "5.3.3" ]; then
+  # No mod_openssl in Travis 5.3.3 boxes
+  composer config -g disable-tls true
+  composer config -g secure-http false
+  composer self-update
+  composer global require drush/drush:6.*
+  composer global require drupal/coder:7.*
+else
+  composer global require drush/drush:7.*
+  composer global require drupal/coder
+fi
+
+# Symlink the things
+if [ -f "$HOME/.config/composer/vendor/bin/drush" ]; then
+  sudo ln -s $HOME/.config/composer/vendor/bin/drush /usr/bin/drush
+elif [ -f "$HOME/.composer/vendor/bin/drush" ]; then
+  sudo ln -s $HOME/.composer/vendor/bin/drush /usr/bin/drush
+else
+  echo "Could not find drush"
+  exit 1
+fi
+
+if [ -f "$HOME/.config/composer/vendor/bin/phpcs" ]; then
+  sudo ln -s $HOME/.config/composer/vendor/bin/phpcs /usr/bin/phpcs
+elif [ -f "$HOME/.composer/vendor/bin/phpcs" ]; then
+  sudo ln -s $HOME/.composer/vendor/bin/phpcs /usr/bin/phpcs
+else
+  echo "Did not find phpcs"
+fi
 
 # PHP Copy-Paste Detection installation.
 wget http://alpha.library.yorku.ca/phpcpd.phar
```

Comment:
Could we go for `composer require --dev sebastian/phpcpd` instead of a phar there? Maybe i'm asking too much but since we are moving away of `phars` i felt it could be a positive thing to go for. More info here https://github.com/sebastianbergmann/phpcpd